pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -234,6 +234,11 @@ class Build {
 				context.println "Running test: ${testType}"
 				testStages["${testType}"] = {
 					context.stage("${testType}") {
+						def keep_test_reportdir = buildConfig.KEEP_TEST_REPORTDIR
```

Comment:
the variable name doesn't match usual camelCase semantics?

Reply:
agree, but there's some that are like that, and some that aren't, wasn't sure which convention to follow!